Formula/c/clickhouse-sql-parser.rb

```diff
@@ -0,0 +1,19 @@
+class ClickhouseSqlParser < Formula
+  desc "Writing clickhouse sql parser in pure Go"
+  homepage "https://github.com/AfterShip/clickhouse-sql-parser"
+  url "https://github.com/AfterShip/clickhouse-sql-parser/archive/refs/tags/v0.3.8.tar.gz"
+  sha256 "dda8b9503bf0e71d78fce260dcf3c72b2929580d554a7e4d03bbb4177c63868b"
+  license "MIT"
+  head "https://github.com/AfterShip/clickhouse-sql-parser.git", branch: "master"
+
+  depends_on "go" => [:build, :test]
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "."
+  end
+
+  test do
+    output = shell_output("#{bin}/clickhouse-sql-parser -v 2>&1")
+    assert_match "v#{version}", output
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
@carlocab Sure, thanks for your correction.